README.md

```diff
@@ -180,3 +196,32 @@ scripts if requested.
 
 There are several tools trying to solve similar problems as Jujutsu. See
 [related work](https://martinvonz.github.io/jj/latest/related-work) for details.
+
+## Contributing
+
+We welcome outside contributions, and there's plenty of things to do, so
+don't be shy. Please ask if you want a pointer on something you can help with,
+and hopefully we can all figure something out.
+
+We do have [a few policies and
+suggestions](https://martinvonz.github.io/jj/prerelease/contributing/)
+for contributors. The broad TL;DR:
+
+- Have a working Rust toolchain.
```

Comment:
Perhaps replace this with "Bug reports are welcome!"? I think bug reports should be classified as contributions, and don't require a toolchain.